### PR TITLE
[MIG] medical_family: Migrated to 10.0

### PR DESCRIPTION
**Pull request by rigo1985**
[MIG] medical_family: Migrated to 10.0

### CODE REVIEW

**Comment by lasley** [comment]
@kellylougheed please review

**Review by kellylougheed — APPROVED** [review]
LGTM

File: medical_family/models/medical_family.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #
 ###############################################################################
```
Comment (lasley) [inline medical_family/models/medical_family.py:21]: Please switch to shorter header styles - https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: medical_family/views/medical_family_view.xml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<openerp>
-    <data>
+<odoo>
 
         <record id="medical_family_view_tree" model="ir.ui.view">
```
Comment (lasley) [inline medical_family/views/medical_family_view.xml:4]: Decrease indent level by four spaces

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks @rigo1985 - comments inline. Also, the module is not currently installable, so it is not working functionally. Please update the `installable` key in the manifest to `True`

**Comment by lasley** [comment]
Sorry but I am closing this PR. The module here was part of #183 & the original copyright cannot be validated. As such, the module will need to be recreated in its entirety instead of just upgraded. Sorry for the inconvenience

**Pull request closed without merging**